lib/scope.js

```diff
@@ -552,17 +540,16 @@ module.exports = {
       if ( $(`#dasigpage`).length > 0 && $(`#dasigpage`)[0].name ) {
         field_data.selector = `#daquestion canvas`;
         field_data.tag = `canvas`;
-        field_data.values = [ `/sign` ];  // Our custom value for signatures. TODO: discuss
-        field_data.var_names = [ page_data.sought_var ];
-        let canvas_pattern = await scope.getSoughtVarPatterns( scope, {
-          field_name: page_data.sought_var,
-        });
-        field_data.name_patterns = [ canvas_pattern ];
-        temp_fields.push( field_data );
+        field_data.rows = [{
+          var: page_data.sought_var,
+          value: '/sign',  // Our custom value for signatures. TODO: discuss
+          checked: '',
+        }];
+        page_data.fields.push( field_data );
```

Comment:
This is for the signature field/canvas. Just restoring it to almost what it was before.